apps/bfd-model/bfd-model-rif/src/test/java/gov/cms/bfd/model/rif/schema/DatabaseSchemaManagerTest.java

```diff
@@ -1,6 +1,6 @@
 package gov.cms.bfd.model.rif.schema;
 
-import org.hsqldb.jdbc.JDBCDataSource;
```

Comment:
My comment was in reference to this.  I believe you've stated that you were going to let both the new and old database types be options for the people running the tests (because the new option increases build times). If someone chooses to use the old option, wouldn't this import be required?

Reply:
Ahh, now I see what you meant -- thanks for clarifying!

Imports are used at compile time, not runtime or test-time. Because the class isn't referenced directly by the code in this file any more, the import isn't needed. I'd suggest you go read more about Java imports here when you have a moment: https://docs.oracle.com/javase/tutorial/java/package/usepkgs.html